docker/compose/aptos-node/haproxy.cfg

```diff
@@ -41,25 +41,20 @@ frontend fe-validator
     tcp-request connection silent-drop if { src -n -f /usr/local/etc/haproxy/blocked.ips }
 
     # We deem a connection rate high when an IP is attempting to reconnect more than twice a min
-    acl ip_high_conn_rate sc0_conn_rate gt 2
+    acl ip_high_conn_rate sc0_conn_rate gt 12
```

Comment:
I think we should make this value very high for now and tune it down later

Reply:
This doesn't blacklist, all connections will converge inside a couple seconds.
Making this value high just opens the flood gates. With current setting you can spam 6180 just using lt 64 IPs.